aptos-move/aptos-vm/src/errors.rs

```diff
@@ -14,26 +14,27 @@ use move_deps::{
 /// These errors are only expected from the module that is registered as the account module for the system.
 /// The prologue should not emit any other error codes or fail for any reason, doing so will result
 /// in the VM throwing an invariant violation
-pub const EACCOUNT_FROZEN: u64 = 1000; // sending account is frozen
-pub const EBAD_ACCOUNT_AUTHENTICATION_KEY: u64 = 1001; // auth key in transaction is invalid
-pub const ESEQUENCE_NUMBER_TOO_OLD: u64 = 1002; // transaction sequence number is too old
-pub const ESEQUENCE_NUMBER_TOO_NEW: u64 = 1003; // transaction sequence number is too new
-pub const EACCOUNT_DOES_NOT_EXIST: u64 = 1004; // transaction sender's account does not exist
-pub const ECANT_PAY_GAS_DEPOSIT: u64 = 1005; // insufficient balance (to pay for gas deposit)
-pub const ETRANSACTION_EXPIRED: u64 = 1006; // transaction expiration time exceeds block time.
-pub const EBAD_CHAIN_ID: u64 = 1007; // chain_id in transaction doesn't match the one on-chain
-pub const ESCRIPT_NOT_ALLOWED: u64 = 1008;
-pub const EMODULE_NOT_ALLOWED: u64 = 1009;
-pub const EINVALID_WRITESET_SENDER: u64 = 1010; // invalid sender (not aptos root) for write set
-pub const ESEQUENCE_NUMBER_TOO_BIG: u64 = 1011;
-// This error code has been deprecated and just left here sp future error codes would not
-// accidentally reuse.
-#[allow(dead_code)]
-pub const RESERVED_UNUSED: u64 = 1012;
-pub const ESECONDARY_KEYS_ADDRESSES_COUNT_MISMATCH: u64 = 1013;
-pub const ESEQ_NONCE_NONCE_INVALID: u64 = 1014;
+// Auth key in transaction is invalid.
+pub const EBAD_ACCOUNT_AUTHENTICATION_KEY: u64 = 1001;
+// Transaction sequence number is too old.
+pub const ESEQUENCE_NUMBER_TOO_OLD: u64 = 1002;
+// Transaction sequence number is too new.
+pub const ESEQUENCE_NUMBER_TOO_NEW: u64 = 1003;
+// Transaction sender's account does not exist.
+pub const EACCOUNT_DOES_NOT_EXIST: u64 = 1004;
+// Insufficient balance (to pay for gas deposit).
```

Comment:
Are these supposed to be triple `/`?

Reply:
No they're not used to generate error maps. Just for reading code.